src/main/resources/application.properties

```diff
@@ -1 +1,17 @@
 spring.application.name=FlightDetailsAPI
+
+#Logging
+logging.level.org.springframework=info
+
+#Database
+spring.datasource.url=jdbc:mysql://localhost:3306/flightdb
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.datasource.username = root
```

Comment:
Make datasource username and password as runtime args.

Reply:
1. Used jasypt to encrypt the datasource username and password in application.properties file. 

2. The algorithm used is the default algorithm (PBEWithMD5AndDES).